app/models/git/repository.rb

```diff
@@ -39,6 +39,11 @@ def read_json_blob(commit, path)
       JSON.parse(raw, symbolize_names: true)
     end
 
+    def read_text_blob(commit, path)
+      oid = find_file_oid(commit, path)
+      read_blob(oid, nil)
```

Comment:
```suggestion
      read_blob(oid, "")
```

Probably better to make this method ensure it returns a string. As we should use `present?` everywhere, it shouldn't hurt us, but means we don't accidently call string methods on `nil`.